api.yaml

```diff
@@ -52,44 +52,59 @@ paths:
       responses:
         "200":
           description: "Success."
+        "400":
+          description: "Invalid input"
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Error'
       x-transportd:
         backend: app
-  /{name}:
-    get:
-      description: An API Call.
-      parameters:
-        - name: "name"
-          in: "path"
-          description: "Name of person to greet."
+  /notification:
+      post:
+        description: Get scanned assets and vulnerabilities from last scan for given site and publish them onto an event stream.
+        requestBody:
           required: true
-          schema:
-            type: "string"
-      responses:
-        "200":
-          description: "Success"
           content:
             application/json:
               schema:
-                $ref: '#/components/schemas/Greeting'
-      x-transportd:
-        backend: app
-        enabled:
-#          - "metrics"
-          - "accesslog"
-          - "requestvalidation"
-          - "responsevalidation"
-          - "lambda"
-        lambda:
-          arn: "name_or_arn"
-          async: false
-          request: '{}'
-          success: '{"status": 200, "bodyPassthrough": true}'
-          error: '{"status": 500, "bodyPassthrough": true}'
+                $ref: '#/components/schemas/ScanNotification'
+        responses:
+          "202":
+            description: "Accepted"
+        x-transportd:
+          backend: app
+          enabled:
+  #          - "metrics"
```

Comment:
This should be uncommented

Reply:
Yep, thanks. I'll uncomment all of the stats stuff to enable it.